packages/grenier/grenier.0.12/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+maintainer: "Frederic Bour <frederic.bour@lakaban.net>"
+authors: "Frederic Bour <frederic.bour@lakaban.net>"
+homepage: "https://github.com/let-def/grenier"
+bug-reports: "https://github.com/let-def/grenier"
+license: "ISC"
+dev-repo: "git+https://github.com/let-def/grenier.git"
+doc: "https://let-def.github.io/grenier/doc"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04"}
+  "dune" {>= "1.2.0"}
+  "stdlib-shims"
```

Comment:
Actually it seems that this library is not needed as the package already requires OCaml >= 4.08:
```
#=== ERROR while compiling grenier.0.12 =======================================#
# context              2.0.8 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(https://github.com/let-def/grenier/releases/download/v0.12/grenier-v0.12.tbz)
# path                 ~/.opam/4.06/.opam-switch/build/grenier.0.12
# command              ~/.opam/4.06/bin/dune build -p grenier -j 47
# exit-code            1
# env-file             ~/.opam/log/grenier-16-2fbc47.env
# output-file          ~/.opam/log/grenier-16-2fbc47.out
### output ###
#       ocamlc balmap/.balmap.objs/byte/balmap__Map.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -40 -g -bin-annot -I balmap/.balmap.objs/byte -I /home/opam/.opam/4.06/lib/stdlib-shims -I baltree/.grenier_baltree.objs/byte -no-alias-deps -open Balmap -o balmap/.balmap.objs/byte/balmap__Map.cmo -c -impl balmap/map.ml)
# File "balmap/map.ml", line 140, characters 12-23:
# Error: Unbound module Int
```
@let-def could you re-release by either increasing the minimal requirement for OCaml to 4.08 or changing the code that use the `Int` module? Also could you at the same time return the `{pinned} --> {dev}` fix?

Reply:
Thanks @mseri and @kit-ty-kate. I have updated the constraints, only the floating point bugs remain.